mne/io/__init__.py

```diff
@@ -39,8 +39,8 @@
 from .eeglab import read_raw_eeglab, read_epochs_eeglab
 
 # for backward compatibility
-from .fiff import RawFIF
-from .fiff import RawFIF as Raw
+from .fiff import Raw
+from .fiff import Raw as RawFIF
```

Comment:
@agramfort almost everywhere in the package (docs, examples, code docstring, etc.) we talk about `mne.io.Raw` not `mne.io.RawFIF`. To make my documentation above about `:func:mne.io.Raw` work properly, I needed to make `RawFIF` the alias of `Raw` instead of the other way around. So I figured this was a reasonable place to put in this change, but I can take it out if you want.